torch_xla/csrc/xla_lower_util.cpp

```diff
@@ -504,6 +504,24 @@ xla::XlaOp BuildDropout(xla::XlaOp input, float probability, xla::XlaOp seed) {
   return input * mask;
 }
 
+std::vector<xla::XlaOp> BuildNativeDropout(xla::XlaOp input, float probability,
+                                           c10::optional<bool> train,
+                                           xla::XlaOp seed) {
+  const xla::Shape& shape = ShapeHelper::ShapeOfXlaOp(input);
+  if (!train.has_value() || *train) {
+    xla::XlaOp prob =
+        XlaHelpers::ScalarBroadcast<float>(probability, shape, input.builder());
+    xla::XlaOp mask = BuildBernoulli(prob, seed, shape.element_type());
+    if (probability > 0.0f) {
+      mask = mask / prob;
```

Comment:
We should scale the activation by `1/(1-p)`. 

Let's do
```
xla::XlaOp one = xla::One(input.builder(), shape.element_type());
mask = mask / (one - prob);
```

Reply:
You are right, that matches the implementation: https://github.com/pytorch/pytorch/blob/f87863335c8e8e5a6f51c6b35194f7b68d44be17/aten/src/ATen/native/Dropout.cpp#L116C1-L118. Fixed!